.github/ISSUE_TEMPLATE/new_vm.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Request a new VM
+about: Assemble the necessary information to request a new VM
+title: ''
+labels: ''
+assignees: ''
+
+---
+### Describe the VM(s) you need
+
+If you do not know the answers to some of these questions, that's okay. Please add the information you do have, and we will work together to gather any other information we need.
+
+Which project are these VMs for?
+- [ ] Project:
+
+How many VMs do you need?
+- [ ] Number of VMs:
+
+What should the FQDN(s) be? (for example, our-new-project-staging1.princeton.edu):
+- [ ] FQDN:
+
+What resources does each VM need?
+- [ ] Number of CPUs per VM:
+- [ ] Amount of memory per VM (in GB):
+- [ ] Amount of storage/disk space per VM (in GB):
+
+What firewall changes should be made? (for example, port 22 for SSH, etc.)
+- [ ] Port number(s)
+- [ ] Service(s) 
+
+What shared resources should be mounted? (for example, Isilon drives)
+- [ ] Resources to mount:
+
```

Comment:
Is it worth including a question whether VMs will need access to the NFS server?

Reply:
Good idea, thanks. I will broaden the scope of that last question to futureproof it, and add NFS server access as an example.